dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/log/LoggerRequestProcessor.java

```diff
@@ -102,19 +92,16 @@ public void process(Channel channel, Command command) {
             case REMOVE_TAK_LOG_REQUEST:
                 RemoveTaskLogRequestCommand removeTaskLogRequest = JSONUtils.parseObject(
                         command.getBody(), RemoveTaskLogRequestCommand.class);
-
-                String taskLogPath = removeTaskLogRequest.getPath();
-
-                File taskLogFile = new File(taskLogPath);
+                List<String> taskLogPaths = removeTaskLogRequest.getPath();
+                OsSystemNativeCommand os = new LinuxSystem();
+                String cmd = "";
+                for (String path : taskLogPaths){
```

Comment:
It is better to split when the length of the taskLogPaths is large, you can do the split in the api module.

Reply:
ds 系统官方推荐目前只是支持 linux 系统，所以使用rm -rf 的方式下应该师符合场景的。
同时也做了未来的扩展可能，加入了系统类的铺垫。
后期在出现多系统支持的时候增加，个人觉得就足够了。

Reply:
I mean can we directly use `Files.delete();` to remove the log file instead of use shell command.
What you concern is using shell command is faster than using api ?

Reply:
这种场景下最适合的应该是一次性删除多文件的方式，而不是一个个删除的方案。
目前没发现在 java JDK 中有任何比较适合的 api 可以直接调用，可以一次性批量删除。

Reply:
OK